runit-java8-bionic/Dockerfile

```diff
@@ -16,6 +16,7 @@ ENV JAVA_TOOL_OPTIONS="-Dcom.sun.management.jmxremote.port=11114 -Dcom.sun.manag
 COPY set_jmx_hostname /etc/my_init.d/set_jmx_hostname
 COPY collectd-jmx.conf /etc/collectd/conf.d/jmx.conf
 
+RUN sed -i -e '/^sleep/a\' -e 'sv stop collectd' /etc/runit/runsvdir/default/collectd/run
```

Comment:
I tested this by running this sed command in a running docker container. It added the line as expected right before exporting LD_PRELOAD. This *should* ensure the /usr/sbin/collectd runs cleanly.